src/pages/Home/HomeTemplate.js

```diff
@@ -51,18 +54,61 @@ class DigitalAssetsGridView extends Component<Props> {
     const filterCount: number = filterOptions.isHideZeroBalance ? 1 : 0
 
     return (
-      <div className='digital-assets-grid-view'>
-        <div className='header'>
-          <div className='container'>
+      <div className={style.home}>
+        <section>
+          <h2 className={style.title}>
+            {t`Transfer`}
+          </h2>
+          <nav className={style.transferButtons}>
+            <JLink
+              className={style.transferButton}
+              href='/send'
+            >
+              <div className={style.transferButtonIcon}>
+                <JIcon
+                  name='home-send-use-fill'
+                  color='blue'
+                />
+              </div>
+              {t`Send`}
+            </JLink>
+            <JLink
+              className={style.transferButton}
+              href='/receive'
+            >
+              <div className={style.transferButtonIcon}>
+                <JIcon
+                  name='home-receive-use-fill'
+                  color='blue'
+                />
+              </div>
+              {t`Receive`}
+            </JLink>
+            <JLink
+              className={style.transferButton}
+              href={JCASH_UTM_URL}
+            >
+              <div className={style.transferButtonIcon}>
```

Comment:
`div` inside `a` is not semantic

Reply:
It is considered ok since HTML 5 spec: https://www.w3.org/TR/2011/WD-html5-20110525/text-level-semantics.html#the-a-element